android/autodispose-android/src/test/java/com/uber/autodispose/android/AutoDisposeAndroidPluginsTest.java

```diff
@@ -20,13 +20,14 @@
 import com.uber.autodispose.android.internal.AutoDisposeAndroidUtil;
 import io.reactivex.functions.BooleanSupplier;
 import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
 import static com.google.common.truth.Truth.assertThat;
 
 public final class AutoDisposeAndroidPluginsTest {
 
-  @After public void tearDown() {
+  @Before @After public void tearDown() {
```

Comment:
tearDown is probably a bad name if you're using this for before and after.

Reply:
Part of Part of 53772ce359b7db21c15342b6edc1ff3d809e35c3